devmand/gateway/src/devmand/test/cli/QueuedCliTest.cpp

```diff
@@ -70,6 +72,29 @@ TEST_F(QueuedCliTest, queuedCli) {
   }
 }
 
+TEST_F(QueuedCliTest, queueOrderingTest) {
+  unsigned int iterations = 20;
+  shared_ptr<CPUThreadPoolExecutor> executor2 =
```

Comment:
executor2 is not the best naming practice

Reply:
I’ll rename it on thursday